roles/network_plugin/calico/tasks/check.yml

```diff
@@ -50,10 +50,14 @@
   run_once: True
   delegate_to: "{{ groups['kube_control_plane'][0] }}"
 
-- name: Check if calico ClusterInformation exists
-  command: "{{ bin_dir }}/calicoctl.sh get ClusterInformation"
-  register: clusterinformation_exists
+- name: Check if calico exists
+  command: "{{ bin_dir }}/calicoctl.sh get ClusterInformation default"
```

Comment:
Do we need to specify `default` here?
I guessed we just need to revert here back to the original which is without `default`.

Reply:
It is necessary.

* the command `calicoctl.sh version` in the next task will also call the `default` ClusterInformation instance.

* the command `calicoctl.sh get ClusterInformation` just list instances of ClusterInformation. It will complete with `ReturnCode 0` even with nothing instance.

* the command `calicoctl.sh get ClusterInformation` which ends with `ReturnCode 0`  means that the CRD `ClusterInformation` exists, do not means that the `default` instance must exist.

* So , we need to use the command  `calicoctl.sh get ClusterInformation default` to make sure the exist of the default instance.

Reply:
Yes, We need to specify `default` here, Otherwise `check version` may still fail.